Fix/sk33lz/vagrant up

sk33lz opened the pull request:
This PR updates the probo-image-builder VM to generate a new docker image call `ubuntu-14.04-lamp-nightly.tar` and export it to the workspace NFS directory after a successful provision.

**To Test**

- [ ] vagrant up

- [ ] check to make sure you have a `ubuntu-14.04-lamp-nightly.tar` file in the `probo-image-builder/workspace` directory.

- [ ] Test inside the Probo VM against a known good repository.

There is still a bug that happens with the `wget -q -O - --header 'Authorization:token <*****>` that @lliss said he is making a different ticket for.

lliss commented on .gitignore:
```diff
@@ -1,10 +1,12 @@
 .tmp
 builder/ubuntu-14.04_lamp.tar
 packer/packer_cache
+packer/scripts/.DS_Store
```
I'm pretty sure you can do a wildcard so you don't have to specify this one separately.

sk33lz replied:
Yeah, I thought about that after the commit...

lliss commented on Vagrantfile:
```diff
@@ -16,7 +16,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # To enable, create a workspace directory in this directory
   if File.directory?("workspace")
-    config.vm.synced_folder "workspace", "/workspace" , type: "nfs"
+    config.vm.synced_folder "workspace", "/vagrant/workspace" , type: "nfs"
```
Why bother doing this? The entire repo is automatically mounted in the vm at `/vagrant`. So if this is causing trouble just remove it and grab the exported stuff out of the directory itself.

sk33lz commented:
I've added a wildcard for the .DS_Store file in .gitignore and reworked my code so that it just exports to the /vagrant directory. Testing again locally.

sk33lz commented:
Saved docker image .tar file is now just in the root of the repo since /vagrant is mounted by default. This should be good to go.
